apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-upload-components/AttachmentsUploadInput.vue

```diff
@@ -193,19 +193,21 @@ export default {
       }
 
       newAttachedFiles.filter(file => !this.attachments.some(f => f.title === file.title)).forEach((newFile,index) => {
-        this.queueUpload(newFile,index);
+        if (this.attachments.length === this.maxFilesCount) {
```

Comment:
what happens for the case when the number of attachments is bigger than maxFileCount ? here you test just for the equality